mifosng-android/src/main/java/com/mifos/api/ApiManager.java

```diff
@@ -0,0 +1,361 @@
+/*
+ * This project is licensed under the open source MPL V2.
+ * See https://github.com/openMF/android-client/blob/master/LICENSE.md
+ */
+
+package com.mifos.api;
+
+import android.util.Log;
+
+import com.google.gson.JsonArray;
+import com.mifos.api.model.ClientPayload;
+import com.mifos.api.model.CollectionSheetPayload;
+import com.mifos.api.model.GpsCoordinatesRequest;
+import com.mifos.api.model.GpsCoordinatesResponse;
+import com.mifos.api.model.Payload;
+import com.mifos.api.model.SaveResponse;
+import com.mifos.api.model.ScorecardPayload;
+import com.mifos.objects.SearchedEntity;
+import com.mifos.objects.User;
+import com.mifos.objects.accounts.ClientAccounts;
+import com.mifos.objects.accounts.loan.LoanApprovalRequest;
+import com.mifos.objects.accounts.loan.LoanRepaymentRequest;
+import com.mifos.objects.accounts.loan.LoanRepaymentResponse;
+import com.mifos.objects.accounts.loan.LoanWithAssociations;
+import com.mifos.objects.accounts.savings.SavingsAccountTransactionRequest;
+import com.mifos.objects.accounts.savings.SavingsAccountTransactionResponse;
+import com.mifos.objects.accounts.savings.SavingsAccountWithAssociations;
+import com.mifos.objects.client.Charges;
+import com.mifos.objects.client.Client;
+import com.mifos.objects.client.Page;
+import com.mifos.objects.client.Savings;
+import com.mifos.objects.db.CollectionSheet;
+import com.mifos.objects.db.OfflineCenter;
+import com.mifos.objects.group.Center;
+import com.mifos.objects.group.CenterWithAssociations;
+import com.mifos.objects.group.Group;
+import com.mifos.objects.group.GroupWithAssociations;
+import com.mifos.objects.noncore.DataTable;
+import com.mifos.objects.noncore.Document;
+import com.mifos.objects.noncore.Identifier;
+import com.mifos.objects.organisation.Office;
+import com.mifos.objects.organisation.ProductSavings;
+import com.mifos.objects.organisation.Staff;
+import com.mifos.objects.survey.Scorecard;
+import com.mifos.objects.survey.Survey;
+import com.mifos.objects.templates.loans.LoanRepaymentTemplate;
+import com.mifos.objects.templates.savings.SavingsAccountTransactionTemplate;
+import com.mifos.services.data.CenterPayload;
+import com.mifos.services.data.ChargesPayload;
+import com.mifos.services.data.GroupPayload;
+import com.mifos.services.data.SavingsPayload;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import retrofit.Callback;
+import retrofit.client.Response;
+import retrofit.mime.TypedFile;
+
+/**
+ * @author fomenkoo
+ */
+public class ApiManager extends BaseApiManager {
+
+    // Basic authentication
+    public void login(String username, String password, Callback<User> callback) {
+        Log.i(getClass().getSimpleName(), "Login - " + username + " Password - " + password);
+        getAuthApi().authenticate(username, password, callback);
+    }
+
+    //
```

Comment:
Please get rid of these comments.

Either use 
```
/**
*
*/
```

or don't add comments at all.